src/utils/analyticsUtils.ts

```diff
@@ -0,0 +1,13 @@
+import { TrackingEventType } from './../models/TrackingEventType';
+import MarketplaceItemModel from './../models/MarketplaceItemModel';
+
+export function trackItemEvent(eventAction: TrackingEventType, item: MarketplaceItemModel): void {
+    if (window && (window as any).dataLayer) {
+        (window as any).dataLayer.push({
+            event: "event",
+            eventCategory: "Link",
+            eventAction: eventAction.toString(),
```

Comment:
If we change the labels (see above) we would probably want to change the event action type.

Reply:
no labels changed;)